src/tool.h

```diff
@@ -97,15 +97,18 @@ struct DigParams
 	{}
 };
 
+DigParams getDigParams(const ItemGroupList &groups,
+		const ToolCapabilities *tp, float time_from_last_punch);
```

Comment:
As mentioned in https://github.com/minetest/minetest/pull/8862/files#r322311388, this is the correct way to specify default values for functions:

```suggestion
		const ToolCapabilities *tp, float time_from_last_punch = 1000000.0f);
```

Now, the two-param overload can be removed.